releasenotes/notes/bosonic-logarithmic-mapper-4b1f24c4ca16cf8b.yaml

```diff
@@ -0,0 +1,17 @@
+---
+prelude: >
+    Up to now, Qiskit Nature could only map bosonic operators to qubit operators using
+    a linear encoding (see :class:`.BosonicLinearMapper`).
```

Comment:
```suggestion
```
The `prelude` section would get rendered at the very top of all release notes and is reserved for overarching announcements.
You should stick to just a `feature` block as done below.